docs/modules/ROOT/pages/csharp.adoc

```diff
@@ -15,17 +15,9 @@ Create or open an existing Virtual Studio project and install Couchbase Lite usi
 
 === Nuget
 
-. Install Couchbase Lite from Nuget (or if you are interested in a prerelease build, install from http://mobile.nuget.couchbase.com/nuget/Developer).
-+
-*Couchbase Lite Community Edition*
-+
-Install the `Couchbase.Lite` package.
-+
-*Couchbase Lite Enterprise Edition*
-+
-If you have a license or you are evaluating the Enterprise edition, install the `Couchbase.Lite.Enterprise` package.
-+
-. Your app must call the relevant `Activate()` function inside of the
+* Add `http://mobile.nuget.couchbase.com/nuget/Developer/` to your Nuget package sources (for prerelease only).
+
+Your app must call the relevant `Activate()` function inside of the
```

Comment:
It's going to easy to forget to update this paragraph when switching between DB to GA in the cycle. Will try to add both.